### DISCUSSION TITLE
Acer SFA16-41 (6800U) - Hangs

### DISCUSSION THREAD

**Discussion by sos-michael** [DavidS95/Smokeless_UMAF#14] [General]
Thanks for even having this idea!

I'm just adding to the list of known 6xxx failures. 

I've tried with both gpt and mbr. Attempts to launch, what looks like 3 repeating messages show up for a split second, and then it just hangs.

Acer use insydeh20 firmwares.

**Comment by sos-michael** [comment]
Just following up with the latest beta. More stuff flashed on the screen but still no go. 

Apparently, there is a log file, but it doesn't seem to detect anything erroneous:

Welcome to SREP (Smokeless Runtime EFI Patcher) 0.1.4c
Opened SREP_Config

Config Size: 0x4E
Parsing Config

Stripping NewLine, Carriage and tab Return

Current Parsing Op LoadFromFS
OP Detected

Commnand LoadFromFS 
Current Parsing SuppressIFPatcher.efi
Found File SuppressIFPatcher.efi 
Current Parsing Op Exec
OP Detected

Commnand Exec 
Current Parsing Op LoadFromFS
OP Detected

Commnand LoadFromFS 
Current Parsing Loader.efi
Found File Loader.efi 
Current Parsing Op Exec
OP Detected

Commnand Exec 
Executing Load from FS

Loaded Image Success -> 64778000
EXEC %r

Executing Load from FS

Loaded Image Success -> 64777000

**Comment by etclub** [comment]
I am also having  the same issue.
I just want to enable s3 sleep mode, so that the power-on cycles of the disk will not increase quickly during sleeping.

**Comment by gr1tz** [comment]
did you found any solution?
